app/Repositories/ReportbackRepository.php

```diff
@@ -179,11 +181,47 @@ public function updateReportbackItems($data)
         foreach ($data as $reportbackItem) {
```

Comment:
Do we want to allow mass-reviewing reportbacks? It doesn't feel super RESTful (which isn't the end of the world), and it also seems like we might be working ourselves into the same corner we were in in Phoenix (where multiple reviewers couldn't safely be working on the same campaign without conflicts).